docs/api/ref/PrivateChannel.md

```diff
@@ -0,0 +1,144 @@
+fdc3.addIntentListener("QuoteStream", async (context) => {
+	const channel: PrivateChannel = await fdc3.createPrivateChannel();
+	const symbol = context.id.ticker;
+
+	// This gets called when the remote side adds a context listener
+	const addContextListener = channel.onAddContextListener((contextType) => {
+		// broadcast price quotes as they come in from our quote feed
+		feed.onQuote(symbol, (price) => {
+			channel.broadcast({ type: "price", price});
+		});
+	});
+
+	// This gets called when the remote side calls Listener.unsubscribe()
+	const unsubscriberListener = channel.onUnsubscribe((contextType) => {
+		feed.stop(symbol);
+	});
+
+	// This gets called if the remote side closes
+	const disconnectListener = channel.onDisconnect(() => {
+		feed.stop(symbol);
+	});
+
+	return channel;
+});
+```
+
+### 'Client-side' example:
+
+```javascript
+try {
+    const resolution3 = await fdc3.raiseIntent("QuoteStream", { type: "fdc3.instrument", id : { symbol: "AAPL"}});
+    try {
+	    const result = await resolution3.getResult();
+        //check that we got a result and that its a channel
+        if (result && result.addContextListener) {
+            const listener = result.addContextListener("price", (quote) => console.log(quote));
+	         result.onDisconnect(() => {
+		          console.warn("Quote feed went down");
+	         });
+
+	         // Sometime later...
+	         listener.unsubscribe();
+        } else {
+            console.warn(`${resolution3.source} did not return a channel`);
+        }
+     } catch(channelError){
+	    console.log(`Error: ${resolution3.source} returned an error: ${channelError}`);
+     }
+} catch (resolverError) {
+    console.error(`Error: Intent was not resolved: ${resolverError}`);
+}
+```
+
+
+## Methods
+
+### `onAddContextListener`
+```ts
+onAddContextListener(handler: (contextType?: string) => void): Listener;
+```
+Adds a listener that will be called each time that the remote app invokes addContextListener on this channel.
+
+Desktop Agents MUST call this for each invokation of addContextListener on this channel, including those that occurred before this handler was registered (to prevent race conditions).
+
+#### See also
+* [`Channel.addContextListener`](Channel#addcontextlistener)
+
+### `onUnsubscribe`
+
+```ts
+onUnsubscribe(handler: (contextType?: string) => void): Listener;
+```
+
+Adds a listener that will be called whenever the remote app invokes `Listener.unsubscribe()` on a context listener that it previously added.
+
+Desktop Agents MUST call this when disconnect() is called by the other party, for each listner that they had added.
+
+#### See also
+* [`Listener`](Types#listener)
+
+### `onDisconnect`
+
+```ts
+onDisconnect(handler: () => void): Listener;
+```
+
+Adds a listener that will be called when the remote app terminates, for example when its window is closed or because disconnect was called. This is in addition to calls that will be made to an onUnsubscribe listeners.
```

Comment:
```suggestion
Adds a listener that will be called when the remote app terminates, for example when its window is closed or because disconnect was called. This is in addition to calls that will be made to onUnsubscribe listeners.
```